src/profile-logic/profile-data.js

```diff
@@ -3320,37 +3320,3 @@ export function nudgeReturnAddresses(thread: Thread): Thread {
     mapForBacktraceSelfStacks
   );
 }
-
-/**
- * Returns true if the thread doesn't include any RefreshDriverTick. This
- * indicates they were not painted to, and most likely idle. This is just
- * a heuristic to help users.
- */
-export function isThreadWithNoPaint({ markers, stringTable }: Thread): boolean {
```

Comment:
It's nice to get rid of this special case as well. I tried with some more profiles and I can't seem to find one that breaks. So I think it's ok to get rid of it right now and consider bringing it back if we find some cases that are not working well with the current code.

Reply:
Thanks for testing! The one thing that came to my mind where looking at markers might be useful is for marker-only profiles. But for those we could also just make the "marker count" a part of the thread score.